main.go

```diff
@@ -38,6 +38,7 @@ var (
 	showVersion         = flag.Bool("version", false, "Print version information.")
 	listenAddress       = flag.String("web.listen-address", ":9091", "Address to listen on for the web interface, API, and telemetry.")
 	metricsPath         = flag.String("web.telemetry-path", "/metrics", "Path under which to expose metrics.")
+	prefixPath          = flag.String("web.external-url", "", "The URL under which PushGateway is externally reachable.")
```

Comment:
Further down, we need to sanitize the flag value in the way it is done [here](https://github.com/prometheus/prometheus/blob/master/cmd/prometheus/config.go#L300).

Nit: Please don't CamelCase Pushgateway. I suggest "...under which the Pushgateway is externally reachable." This is consistent with the wording and capitalization used in the README.md.